.github/workflows/deploy-feed-discovery-image.yaml

```diff
@@ -0,0 +1,26 @@
+name: build-and-push-feed-discovery
+
+on:
```

Comment:
We could probably add a path section in here to isolate this to changes in `src/api/feed-discovery` only.  cc @menghif, who just did this for our docs workflows.